app/controllers/requests_controller.rb

```diff
@@ -23,8 +23,9 @@ def edit
   # POST /requests
   def create
     @request = Request.new(request_params)
-
     if @request.save
+      Mailer.send_generic_email(false, @request.email, 'workshop.portal@gmail.com', I18n.t('requests.email.topic'),
```

Comment:
`workshop.portal@gmail.com` should probably be placed in some kind of configuration file.